templates/dag-deploy/dag-server-auth-sidecar-configmap.yaml

```
@@ -23,15 +23,14 @@ data:
       }
     http {
       upstream astro-dag-server {
-        server localhost:{{ .Values.dagDeploy.ports.dagServerHttp }} ;
+        server 127.0.0.1:{{ .Values.dagDeploy.ports.dagServerHttp }} ;
       }
       server {
         server_name {{ .Release.Name }}.{{ .Values.ingress.baseDomain }} ;
         listen {{ .Values.authSidecar.port }}  ;
-        # Disable Nginx Server version
-        server_tokens off;
-{{ include "default_nginx_settings" . | indent 6 }}
+{{ include "default_nginx_server_settings" . | indent 8 }}
         location  = /auth {
+{{ include "default_nginx_settings" . | indent 8 }}
```

Since this isn't nindent of a data structure, should we just indent it so it aligns with the code, rather than calling `| indent 8` ?
```suggestion
        {{ include "default_nginx_server_settings" . }}
```